static/js/tools/map/place_options.tsx

```diff
@@ -242,6 +242,11 @@ function loadEnclosedPlaces(
           `Sorry, ${place.value.enclosingPlace.name} does not contain places of type ` +
             `${enclosedPlaceType}. Try picking another type or place.`
         );
+        document.getElementById("place-list").innerHTML = "";
```

Comment:
I don't think we want to clear the selected place because the alert came after user selected a place type so it should only clear the place type.

Reply:
Thanks for the comment, I have modified the code such that the place type is only cleared and the `place` is not modified.